### DISCUSSION TITLE
Serverless does create another Cognito pool after delete from same profile

### DISCUSSION THREAD

**Discussion by alakdam07** [serverless/serverless#10861] [Q&A]
I deleted Cognito user-pool from AWS' console. After delete the user-pool, rename the pool name, try to deploy the app. But I am getting serverless  error:   `An error occurred: CognitoUserPool - Updates are not allowed for property - UserPoolName. (Service: AWSCognitoIdentityProvider; Status Code: 400; Error Code: InvalidParameterException; Request ID: null; Proxy: null).` And If I keep old pool name and deployed the app. I am getting this error:  ` Stack:arn:aws:cloudformation:..... is in UPDATE_ROLLBACK_FAILED state and can not be updated.`

**Comment by pgrzesik** [comment]
Hello @alakdam07 - is there a reason why you removed the pool manually and created it via Framework? It is not recommended to do so and there's no guarantee that such approach would work. If you'd like to remove something that was deployed with Framework, make sure to do it via configuration and `deploy`.

**Reply by alakdam07** [reply]
Thank you for quick response 🙏🏾. I did not know that it was bad approach until I deleted the pool. Is there any better approach, if I want to delete any AWS' service after creation.

**Reply by pgrzesik** [reply]
No worries - the best way to remove is to remove the definition of the resource from your configuration and run the deployment again. If you want to remove everything from your service, then you can run `sls remove` command.

**Reply by alakdam07** [reply]
Thank you @pgrzesik.
